.circleci/Dockerfile

```diff
@@ -18,13 +18,6 @@ RUN wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER
 RUN echo -e '#!/bin/bash\njava -jar /usr/local/bin/swagger-codegen-cli.jar "$@"' > /usr/local/bin/swagger-codegen && \
 	chmod +x /usr/local/bin/swagger-codegen
 
-# Install protoc
-ENV PROTOC_VERSION=3.12.0
-RUN wget https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip \
-    -O /protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
-    unzip /protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local/ && \
-    rm -f /protoc-${PROTOC_VERSION}-linux-x86_64.zip
```

Comment:
🎉. This change isn't effective as of this PR, it will be the next time we push the docker image.